src/txn/txn_timestamp.c

```diff
@@ -637,7 +637,7 @@ __wt_txn_set_commit_timestamp(
 			    ts_string[0], ts_string[1]);
 		}
 
-		if (has_stable_ts && commit_ts < stable_ts) {
+		if (has_stable_ts && commit_ts <= stable_ts) {
```

Comment:
update the error msg to reflect the condition.
This condition change is needed for durable timestamp check as well in set_durable_timestamp